scylla-cql/src/types/deserialize/value.rs

```diff
@@ -73,6 +73,49 @@ where
     }
 }
 
+/// Values that may be empty or not.
+///
+/// In CQL, some types can have a special value of "empty", represented as
+/// a serialized value of length 0. An example of this are integral types:
+/// the "int" type can actually hold 2^32 + 1 possible values because of this
+/// quirk. Note that this is distinct from being NULL.
```

Comment:
If it's distinct from null  doesn't it mean that int can hold 2^32 + 2 values (int values + null + empty)?